user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,139 @@
+
+### Register using the Catalog API
+
+#### Authentication
+
+The API call is authenticated using [organization keys](xref:Managing_DCP_keys#organization-keys). The key needs to be added to the HTTP request in a header called **Ocp-Apim-Subscription-Key**.
+
+#### API call signature
+
+The call to register a Catalog item can be found at the following URL: <https://api.dataminer.serivces/api/key-catalog/v1-0/catalog/register>. To call the API you will need to use the **POST** HTTP method.
+
+The call accepts a binary file as input. This file should be a zip file containing a ["manifest.yml"](xref:Register_Catalog_Item#manifest-file) and the item you want to upload.
+
+#### Manifest file
+
+This file will contain all necessary information to register a Catalog item or version. This manifest file should be a valid yml file and will contain "required" and "optional" attributes to add extra information to the Catalog item. Note that limitations may apply to certain attributes based on length or formatting.
+
+```yml
+# [Required]
+# Possible values: 
+#   - AutomationScript: If the catalog item is an Automation Script that can be deployed on a DataMiner System.
+#   - AppPackage: If the catalog item is a Solution Package that can be deployed on a DataMiner System.
+#   - ProtocolPackage: If the catalog item is a Protocol that can be deployed on a DataMiner System.
+#   - Visio: If the catalog item is a Visio that can be deployed on a DataMiner System.
+type: 'AutomationScript'
+
+# [Required] 
+# The id of the catalog item.
+# All versions for the same id are shown together in the catalog.
+# This id can not be changed. 
+# If the id is not filled in, the API will return one and consider this as the registration of a new item.
+# If the id is filled in, a new version will be added to the catalog item with the given version number.
+#   Must be a valid GUID.
+id: '<fill in guid here or leave empty>'
+
+# [Required] 
+# The human-friendly name of the catalog item. 
+# Can be changed at any time.
+#   Max length: 100 characters.
+#   Can not contain newlines.
+#   Can not contain leading or trailing whitespace characters.
+title: 'add-new-switch'
+
+# [Required] 
+# The version in format X.X.X
+# Once a version has been published it can no longer be changed.
+# The version should be incremented to do any changes.
+# If you want to create a package that is not final, you can provide a label at the end: X.X.X-label
+# If a catalog item has a label in the version, then the version is considered to be a pre-release version, and not an official one.
+#   Valid versions range from 0.0.0 to 2147483647.2147483647.2147483647
+#   The label can contain characters a-z0-9 with a max length of 30 characters.
+version: '1.0.0'
```

Comment:
Are we then still supposed to 'tag' a solution? What will happen if you don't change the manifest file, but tag with a new version (or the other way around)?

Reply:
What do you mean with tag a solution?
Versioning is controlled by the manifest file and nothing else. If you do not update is you will get an error telling you this version already exists.